components/gc-featured-link/_base.scss

```
@@ -1,21 +1,24 @@
 /* Provisional featured link component */
 
-.gc-featured-link {
+.gc-featured-link.provisional {
 	background-color: $bg-gctheme;
+	color: #fff;
+	font-family: Lato, sans-serif;
+	opacity: .9;
+	padding-bottom: $space-md;
+	padding-top: $space-md;
 
-	&.provisional {
-		font-family: Lato, sans-serif;
-		opacity: .9;
-		padding-bottom: $space-md;
-		padding-top: $space-md;
+	p:first-of-type {
+		margin-bottom: $space-0;
+	}
 
-		p:first-of-type {
-			font-weight: $bold-weight;
-			margin-bottom: $space-0;
-		}
+	a {
+		color: #fff;
+		font-weight: $bold-weight;
+	}
 
-		p, a {
-			color: #fff;
-		}
+	html:not(.wb-disable) &[data-bg-color] {
+		background-color: transparent;
+		color: #fff;
```

Considérant que notre background est white-smoke je crois que la couleur de text adéquat serait noir et non blanc.

```suggestion
		color: #000;
```

Et un override additionnelle sera requis pour les ancres

```suggestion
		color: #000;
		
		a {
			color: #000;
		}
```